common/src/lib.rs

```diff
@@ -48,6 +48,8 @@ pub enum LogProfile {
     DebugAll,
     /// Logs trace level from all crates to a file
     TraceAll,
+    /// maximum logging for blink
+    TraceBlink,
 }
```

Comment:
You're just adding another log option here so this comment is more generic: typically, people just make use of the `RUST_LOG` env, where you can set different log levels for different crates (and even mods). Was there a reason for going with custom/specific method?

Reply:
To my knowledge, the custom method was done so it would allow routing the logs to the component in uplink vs just outputting it to a file or to terminal

Reply:
To my knowledge they are, but these are more specific to the [logging component](https://github.com/Satellite-im/Uplink/blob/dev/ui/src/components/debug_logger/mod.rs#L13) in uplink.

Reply:
@zeenix  is there a way to make the logger emit the trace logs for one crate and ignore the trace logs for all other crates using environment variables? So far I only know how to use `RUST_LOG` to set the same log level for all crates.

Reply:
> @zeenix is there a way to make the logger emit the trace logs for one crate and ignore the trace logs for all other crates using environment variables?

Yes, sir. As I mentioned, you can even set different levels for even mods. `RUST_LOG=tokio=info,uplink=trace,zbus=debug,zbus::connection=trace`.